src/components/display-rectangle.js

```diff
@@ -0,0 +1,28 @@
+const {DOM, createClass} = require("react");
+const {div, canvas} = DOM;
+const React = require("react");
+
+const DisplayRectangle = createClass({
+    render: function() {
+      const {store} = this.context;
+      const {selNode} = store.getState().stackingContext;
+      var boundingRect = undefined;
```

Comment:
No need to initialize to `undefined` as far as I can tell.

`var boundingRect;` would work just fine, no?